pkg/pod/container.go

```diff
@@ -235,6 +235,29 @@ func (builder *ContainerBuilder) WithEnvVar(name, value string) *ContainerBuilde
 	return builder
 }
 
+// WithVolumeMounts configures pod volumes to mount inside the container.
+func (builder *ContainerBuilder) WithVolumeMounts(volMounts []v1.VolumeMount) *ContainerBuilder {
+	glog.V(100).Infof("Adding VolumeMount(s) to the container's definition")
```

Comment:
Can you add the container name to glog so it is clear?

Reply:
thanks, added